types/google-apps-script/google-apps-script.spreadsheet.d.ts

```diff
@@ -2056,6 +2056,7 @@ declare namespace GoogleAppsScript {
       getFontFamily(): string | null;
       getFontSize(): Integer | null;
       getForegroundColor(): string | null;
+      getForegroundColorObject(): Color;
```

Comment:
TextStyle#getForegroundColorObject returns `Color | null`.

```suggestion
      getForegroundColorObject(): Color | null;
```

Code example:

```js
function test() {
  const color = SpreadsheetApp.newTextStyle().build().getForegroundColorObject(); // null
  Logger.log(color);
}
```

Reply:
I really like your suggestion. Thank you @mtgto. I was pondering how to get a test wrapper to add a test for the method. I will make the necessary changes today and add the test.

Reply:
All done. Also added the setForegroundColorObject method which was also missing (from TextStyleBuilder), and tests for both get and set methods.

Reply:
Thanks! 👍